docs/starters.yml

```diff
@@ -2309,3 +2309,34 @@
     - Optimized images
     - Sitemap Page
     - Seo Ready
+    - url: https://ugglr.github.io/gatsby-clean-portfolio/
+  repo: https://github.com/ugglr/gatsby-clean-portfolio
+  description: A clean themed Software Engineer Portfolio site, showcasing soft skills on the front page, features project card showcases, about page. Responsive through react-bootstrap components together with custom CSS style sheets. SEO configured, just need to add google analytics tracking code. 
+  tags:
+    - Portfolio
+    - SEO
+    - Bootstrap
+    - react-bootstrap
+    - responsive
+    - clean design
+    - Landing page
+    - about page
+    - portfolio page
+    - social linking
+  features:
+    - Resume
+    - CV
+    - google analytics
+    - easy favicon swap
+    - Gatsby SEO plugin
+    - Clean layout
+    - White theme
+    - grid using react-bootstrap
+    - bootstrap4 classes available
+    - font-awesome Library for icons
+    - Portfolio site for developers
+    - custom project cards
+    - easily extendable to include blog page
+    - Responsive design
+
+
```

Comment:
Only a single ending blank line is required, not two 🤓

Reply:
Great catch! Applying change